Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/DeclarationsSectionTranslator.swift

```diff
@@ -95,13 +186,16 @@ struct DeclarationsSectionTranslator: RenderSectionTranslator {
                     for alternateDeclaration in decls {
                         let renderedTokens = alternateDeclaration.declarationFragments.map(translateFragment)
 
-                        let platformNames = platforms
+                        let platformNames =
+                            platforms
                             .compactMap { $0 }
                             .sorted(by: \.rawValue)
```

Comment:
Minor: This "platformNames" value is the same for every alternative declaration in `decls`. You could move it out of the loop and compute it once.

Reply:
Question: Why are `nil` platforms filtered out for alternative declarations 
```swift 
let platformNames = platforms
    .compactMap { $0 }
    .sorted(by: \.rawValue)
```
but not for the main declaration
```swift
let platformNames = platforms.sorted { (lhs, rhs) -> Bool in
    guard let lhsValue = lhs, let rhsValue = rhs else {
        return lhs == nil
    }
    return lhsValue.rawValue < rhsValue.rawValue
}
```
?

Reply:
Not sure. 😅 I think this is a situation of "old code versus new code". I'll have to take a closer look to see what the semantics should be.

Reply:
It looks like the original PR for the alternate-declarations support (https://github.com/apple/swift-docc/pull/654) tried to copy/paste the sort function from the main declaration, but the current alternative was suggested and i applied it without considering that it dropped nil platforms. I'll extract the sort function used for the main declaration into a version that the alternate declarations can use as well.